Content/default/src/Server/Server.fs

```diff
@@ -32,11 +30,7 @@ let todosApi = {
         }
 }
 
-let webApp =
-    Remoting.createApi ()
-    |> Remoting.withRouteBuilder Route.builder
-    |> Remoting.fromValue todosApi
-    |> Remoting.buildHttpHandler
+let webApp = Api.make todosApi
```

Comment:
I would prefer Route.builder to be a required parameter, or for there to be an overload accepting it

Reply:
It's an optional parameter on `Api.make`. It defaults to `/api/{api name}/{api method}` which is a good starting point and most people tend to stick with.

Reply:
We should raise issues on the API design inside SAFE.Meta though.

Reply:
Great!